modules/openapi-generator/src/main/resources/tiny-cpp-client/helpers-header.mustache

```diff
@@ -0,0 +1,24 @@
+#ifndef _HELPERS_H_
```

Comment:
Same as with double underscore - underscore and upper character combination is reserved by C++ standard.

Also I think you could switch to `#pragma once`